deploy/helm/sumologic/values.yaml

```diff
@@ -3999,6 +3999,10 @@ metadata:
       #   - name: es-certs
       #     mountPath: /certs
       #     readOnly: true
+      # extraPorts:
+      #   - name: otlphttp2
+      #     containerPort: 4319
+      #     protocol: TCP
```

Comment:
Please add documentation entry about this key in `deploy/helm/sumologic/README.md`